.azuredevops/pipelines/publish-test-results.yml

```diff
@@ -0,0 +1,81 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+trigger:
+- master
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: |
+    python -m pip install --upgrade pip setuptools wheel
+    pip install ghapi
+  displayName: 'Install Python tools'
+
+- task: PythonScript@0
+  displayName: 'Download test artifacts'
```

Comment:
Shot in the dark, but I'd be curious if you could just use the Download Pipeline/Build Artifacts AzDO task: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-pipeline-artifact?view=azure-devops or https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-build-artifacts?view=azure-devops

The only reason I suggest that is that both github actions and azure devops artifacts use the same backend, I'd be curious if it just works. Though I don't know what the "pipeline" name would be.

Besides that, I can't think of anything else that would work better. Maybe reaching out to someone who works on Azure DevOps may be the next best step.

Reply:
Thanks for the suggestion! I took a look at the [task definition](https://github.com/microsoft/azure-pipelines-tasks/blob/e6aab1600c8d50e44518ae0de54995f2716b66ad/Tasks/DownloadBuildArtifactsV0/task.json) for that and it's a wrapper for the TFS HTTP endpoint, so it doesn't look like it's possible to get through that to the underlying shared backend with GitHub. I'll see if I can find anyone on AzDevOps who might be able to help.